pandas/tests/frame/test_timeseries.py

```diff
@@ -601,3 +601,35 @@ def test_frame_to_period(self):
         tm.assert_index_equal(pts.columns, exp.columns.asfreq('M'))
 
         pytest.raises(ValueError, df.to_period, axis=2)
+
+
+class TestDataFrameTimestamps(object):
+    def test_map_box_timestamps(self):
+        # GH#2689, GH#2627
+        s = Series(date_range('1/1/2000', periods=10))
+
+        def f(x):
+            return (x.hour, x.day, x.month)
+
+        # it works!
+        s.map(f)
+        s.apply(f)
+        DataFrame(s).applymap(f)
+
+    def test_frame_setitem_timestamp(self):
+        # GH#2155
+        columns = DatetimeIndex(start='1/1/2012', end='2/1/2012',
+                                freq=offsets.BDay())
+        index = lrange(10)
+        data = DataFrame(columns=columns, index=index)
+        t = datetime(2012, 11, 1)
+        ts = Timestamp(t)
+        data[ts] = np.nan  # works
+
+    def test_compare_invalid(self):
+        # GH 8058
+        df = DataFrame(np.random.randn(5, 2))
+        a = df[0]
+        b = Series(np.random.randn(5))
```

Comment:
this is a series test

Reply:
OK.  Any complaint if I change the DataFrame call to use Series instead?